package.json

```diff
@@ -24,14 +24,15 @@
   },
   "dependencies": {
     "@babel/core": "^7.12.10",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.14.5",
+    "@babel/plugin-proposal-optional-chaining": "^7.14.5",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
     "@babel/preset-env": "^7.12.11",
     "@babel/preset-react": "^7.12.10",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.4.3",
     "babel-loader": "^8.2.2",
     "browserslist": "^4.16.1",
     "browserslist-useragent-regexp": "^2.1.1",
-    "caniuse-lite": "^1.0.30001178",
```

Comment:
Running `npx browserslist@latest --update-db` seems to have removed this dependency... 🤷‍♂️